modal/_runtime/user_code_imports.py

```diff
@@ -66,20 +58,28 @@ def construct_webhook_callable(
     # For webhooks, the user function is used to construct an asgi app:
     if webhook_config.type == api_pb2.WEBHOOK_TYPE_ASGI_APP:
         # Function returns an asgi_app, which we can use as a callable.
+        from modal._runtime.asgi import asgi_app_wrapper
```

Comment:
Nit, but it looks like we use this module in every branch (or raise an error) so could import it as `asgi` once at the top of the function?

Reply:
yeah good point, will fix